pyspecProcScripts/first_level/QESR_rescale.py

```diff
@@ -0,0 +1,61 @@
+from pyspecdata.datadir import pyspec_config # piggyback on _pyspecdata
+from pint import UnitRegistry
+from numpy import sqrt
+class calib_info (object):
+    "class for caching calibration info"
+    def __init__(self):
+        self.current_calib_name = None
+        self.current_diam_name = None
+    def use_calibration(self, calibration_name):
+        if calibration_name != self.current_calib_name:
+            assert type(calibration_name) is str, "the calibration name must be specified as a string!"
+            assert len(calibration_name)>0, "you MUST use a calibration name and set the values `[calibration_name] q` and `[calibration_name] propFactor` in your _pyspecdata file!"
+            self.current_calib_name = calibration_name
+            # note that this will fail if you don't have "[calibration_name] propFactor" and "[calibration_name] q" set in your pyspecdata file
+            self.default_Q = float(pyspec_config.get_setting(f"{calibration_name} Q"))
+            self.dint_propFactor = float(pyspec_config.get_setting(f"{calibration_name} Proportionality factor"))
+    def use_diameter(self, diameter_name):
+        if diameter_name != self.current_diam_name:
+            assert type(diameter_name) is str, "the diameter name must be specified as a string!"
+            assert len(diameter_name)>0, "you MUST use a diameter name and set the values `[diameter_name] q` and `[diameter_name] propFactor` in your _pyspecdata file!"
+            self.current_diam_name = diameter_name
+            self.d = float(pyspec_config.get_setting(f"{diameter_name} diameter"))
```

Comment:
pulls the proportionality factor, Q, and diameter of capillary from _pyspecdata config file